### PR TITLE
Prepare release 1.29.0

### PR DESCRIPTION
**Pull request by lmolkova**
- update spec version to 1.39.0
- add link to otel.io workflow to check links before releasing to contributing guide
- common release things (schema_next, changelog)

### CODE REVIEW

**Review by maryliag — APPROVED** [review]

File: CHANGELOG.md
Comment (trask) [inline CHANGELOG.md:27]: unrelated to this PR, it would be nice if these were links

File: CONTRIBUTING.md
Comment (trask) [inline CONTRIBUTING.md:370]: will this run against the (existing) semantic convention repo tag that's pinned currently in the website repo?

https://github.com/open-telemetry/opentelemetry.io/blob/a025c25aaf1aef653caa34e49e8714472cbeddbd/.gitmodules#L19-L22

File: model/database/common.yaml
Comment (trask) [inline model/database/common.yaml:40]: ❤️

**Review by trask — APPROVED** [review]

File: CONTRIBUTING.md
Comment (lmolkova) [reply CONTRIBUTING.md:370]: yep, it will run against any ref that you provide

image

File: CHANGELOG.md
Comment (lmolkova) [reply CHANGELOG.md:27]: absolutely!

https://github.com/open-telemetry/semantic-conventions/issues/1627

**Comment by lmolkova** [comment]
still need an approval from @open-telemetry/specs-semconv-approvers

File: CONTRIBUTING.md
Comment (trask) [reply CONTRIBUTING.md:370]: oh, nice!

**Review by joaopgrassi — APPROVED** [review]
Thanks!

**Pull request merged**
